build_tools/benchmarks/mmperf/run_mmperf.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Runs SHARK tank using both SHARK-Runtime and IREE-Runtime, producing benchmark
+# numbers.
+#
+# Usage:
+#    ./run_shark.sh \
+#        <SHA of https://github.com/nod-ai/SHARK.git to pin to> \
+#        <pytest regex> e.g. "cpu", "cuda", "cuda and torch".
+#        <driver> e.g. "cpu", "cuda", "vulkan"
+#        <output directory>
+
+set -xeuo pipefail
+
+export REPORT_DIR=$1
+# Either `cpu` or `cuda`.
+export BACKEND=$2
+
+git clone https://github.com/mmperf/mmperf.git
+cd mmperf
+
+# Update IREE.
+cd external/iree
+git fetch --all
+git checkout origin/main
+git submodule update --init --recursive
+cd -
```

Comment:
nit: I prefer `pushd` and `popd` over `cd -` (we use those in a few scripts)

I wasn't actually aware that `cd` stored state in the `OLDPWD` env var before seeing this `cd -` and wondering what it does. At least with `pushd` and `popd`, the stateful (stack-based) nature of the commands is more obvious.

Reply:
Sounds good. I was also in two minds about this but glad there's a preference.